WordPress/Sniffs/VIP/SlowDBQuerySniff.php

```diff
@@ -50,6 +51,10 @@ public function getGroups() {
 	 */
 	public function process_token( $stackPtr ) {
 
+		if ( $this->has_whitelist_comment( 'slow query', $stackPtr ) ) {
+			return;
+		}
+
 		if ( $this->has_whitelist_comment( 'tax_query', $stackPtr ) ) {
 			return;
```

Comment:
@jrfnl Thanks for your help. I’ll go ahead and include the warning in this pull request, it would be great if you could take a look whenever you find the time.